src/go/rpk/pkg/cli/cmd/cluster/config/edit/edit.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package edit
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cluster/config/export"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cluster/config/importconfig"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func NewCommand(fs afero.Fs, all *bool) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit",
+		Short: "Edit cluster configuration.",
+		Long: `Edit cluster-wide configuration properties
+
+Cluster-wide properties apply to all nodes in a redpanda cluster.  For
+node properties, use the 'rpk config' command.
+`,
+		Args: cobra.ExactArgs(0),
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			client, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			// GET the schema
+			schema, err := client.ClusterConfigSchema()
+			out.MaybeDie(err, "unable to query config schema: %v", err)
+
+			// GET current config
+			var current_config admin.Config
+			current_config, err = client.Config()
+
+			// Generate a yaml template for editing
+			file, err := ioutil.TempFile("/tmp", "config_*.yaml")
+			out.MaybeDie(err, "unable to create temporary file: %v", err)
```

Comment:
Nit: logs should probably all start with a capital letter.

Reply:
Just did a quick grep for MaybeDie and it looks like most of the existing calls user a lowercase letter too

Reply:
My preference is for fatal errors (anything that dies) to quit with a lowercase, following the idiom that errors are lowercase. Visually to me it's clearer that when a program exits with this, it's an error, vs. some normal state that I read through and then understand things aren't right. But, preferences vary here.